tests/scripts/analyze_outcomes.py

```diff
@@ -542,8 +542,38 @@ def do_analyze_driver_vs_reference(results: Results, outcomes: Outcomes, args) -
             'ignored_suites': [
                 # Skipped in the accelerated component
                 'aes', 'aria', 'camellia',
+                # These require AES_C and CAMELLIA_C to be enabled in order for the cipher
+                # module (actually cipher_wrapper) to work properly. However these symbols
+                # are disabled in the accelerated component so we ignore them.
```

Comment:
Ok, that will be something interesting to include in the user documentation: in those builds when we say GCM and CCM work, we mean using the `gcm.h`/`ccm.h` API or the PSA Crypto API, but not the `cipher.h` API. (Nothing to change here, just making a note.)